hr_timesheet_name_customer/views/hr_timesheet_name_customer_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr_timesheet_name_customer_view" model="ir.ui.view">
```

Comment:
Please follow [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#inherited-xml) when for inherited xml ids